README.md

```diff
@@ -99,6 +105,18 @@ The default path is `/tmp/redhatDependencyAnalyticsReport.html`.
 **Golang** :
 * `useGoMVS` : Use the minimal version selection algorithm to select a set of module versions to use when building Go packages.
 
+**HTTP Proxy** :
+* `httpProxy` : Configure HTTP proxy settings for the extension. There are three options available:
+  - `on`: Always use the HTTP proxy regardless of VS Code's proxy settings
+  - `off`: Never use the HTTP proxy regardless of VS Code's proxy settings
+  - `fallback`: Use VS Code's proxy settings (default behavior)
+
+**Maven and Gradle Wrapper** :
+* `mavenWrapper` : Configure whether to use Maven or Gradle wrapper. There are three options available:
+  - `on`: Always use the wrapper regardless of VS Code's `maven.preferMavenWrapper` setting
+  - `off`: Never use the wrapper regardless of VS Code's `maven.preferMavenWrapper` setting
+  - `fallback`: Use VS Code's `maven.preferMavenWrapper` setting (default behavior)
```

Comment:
How accurate to the config should this be? the shape of the config is:
```json
"redHatDependencyAnalytics": {
    "mvn": {
        "preferWrapper": "true/false/fallback"
    },
    "gradle": {
         "preferWrapper": "true/false/fallback"
    }
}
```
Which reminds me, I forgot to actually add gradle wrapper support into vsce :no_mouth:

Reply:
SHould we remove the gradle wrapper?

Reply:
It was added. Thanks Noah